cs3api4lab/api/cs3_file_api.py

```diff
@@ -68,7 +68,18 @@ def stat(self, file_id, endpoint=None):
             }
 
         self.log.info('msg="Failed stat" fileid="%s" reason="%s"' % (file_id, stat_info.status.message))
-        raise FileNotFoundError(stat_info.status.message + ", file " + file_id)
+        return {
```

Comment:
Does this cause an issue in the frontend?

Reply:
Yes, it prevented shares from rendering

Reply:
And not having a return of 404 doesn't break anything else?

Reply:
No, it just ommits the empty share when rendering created/received shares. I'll come up with something more clear with Api unification PR. This is part of a bigger question, i. e. how to handle moving/renaming/deleting shared files, because shares are indepentent entities from the files. I need to think about it. But let's focus on this now.